
Temporary fix to get the PDF Download extension of Mozilla / Firefox to
recognize the PDF.

  - Rene Rebe <rene@exactcode.de>

--- cvs/archivista/webclient/perl/inc/Frames/TableFrame.pm.vanilla	2005-09-21 16:59:55.000000000 +0000
+++ cvs/archivista/webclient/perl/inc/Frames/TableFrame.pm	2005-09-21 13:17:10.000000000 +0000
@@ -487,7 +487,7 @@
 		
 		if ($displaySourceFile == 1) {
 		  if ($$hash_ref->{QuelleIntern}) {
-		    my $url = "$$cgi_dir/frame.pl?source=1&akte=$thisAkte"; 
+		    my $url = "$$cgi_dir/frame.pl?source=1&akte=$thisAkte&x=internal.pdf"; 
           $string .= qq{<td style="padding-left: 10px;"><a href="$url">Download</td>};
 		  } else {
         $string .= qq{<td style="padding-left: 10px;">&nbsp;</td>};
